runtime-modules/bureaucracy/src/constraints.rs

```diff
@@ -0,0 +1,40 @@
+use codec::{Decode, Encode};
```

Comment:
Can we instead use a version of this from the common module that you added?

Reply:
Sure!